uptane-standard.md

```diff
@@ -295,22 +304,20 @@ The overarching goal of Uptane is to provide a system that is resilient in the f
 
 We assume that attackers may want to achieve one or more of the following goals, in increasing order of severity:
 
-* Read the contents of updates to discover confidential information or reverse-engineer firmware
+* Read the contents of updates to discover confidential information, reverse-engineer firmware, or compare two firmware images to identify security fixes and hence determine the fixed security vulnerability
 * Deny installation of updates to prevent vehicles from fixing software problems
 * Cause one or more ECUs in the vehicle to fail, denying use of the vehicle or of certain functions
-* Control the vehicle or ECUs within the vehicle
+* Control ECUs within the vehicle, and possibly the vehicle itself
 
 ## Attacker capabilities {#capabilities}
 
 Uptane is designed with resilience to compromise in mind. We assume that attackers may develop one or more of the following capabilities:
 
-* Read and analyze the contents of previous and/or current versions of software, as well as the update sequence and instructions
 * Intercept and modify network traffic (i.e., perform man-in-the-middle attacks). This capability may be developed in two domains:
     * Outside the vehicle, intercepting and modifying traffic between the vehicle and software repositories
     * Inside the vehicle, intercepting and modifying traffic on one or more vehicle buses (e.g. via an OBD port or using a compromised ECU as a vector)
-* Compromise and control one or more ECUs within a vehicle
-* Compromise signing or encryption keys
-* Compromise and control software repository servers (and any keys stored on the repository)
+* Compromise and control either a director repository or image repository server, and any keys stored on the repository, but not both the director and image repositories. 
+* Compromise either a primary ECU or a secondary ECU, but not both in the same vehicle
```

Comment:
I'm not fully convinced that it's a good idea to remove the worst-case scenarios, but I'm willing to go with it. In any case, all of the items in this list should either end with a period or all of them should not.